code/__DEFINES/mob_defines.dm

```diff
@@ -35,8 +35,8 @@
 #define MOB_SPIRIT		(1 << 9)
 #define MOB_PLANT		(1 << 10)
 
-#define AGE_MIN 17			//youngest a character can be
```

Comment:
Tweaking the max is fine, but for the love of god please keep this as 17. 

I know IPCs aren't biological and chassis age can be a different thing, but please, keep this bar where it is. Given the kind of art *some* players get I don't want **any** leeway in how people portray characters. 

TLDR - This opens a can of worms I ain't prepared to deal with.

Reply:
100% Agree, the AGE_MIN needs to be maintained at 17

Reply:
Genuine question, why is this 17 instead of 18?